app/controllers/api/v1/migrations/external_controller.rb

```diff
@@ -83,8 +83,8 @@ def create_role
         def create_user
           user_hash = user_params.to_h
 
-          # Re-write LDAP and Google to greenlight
-          user_hash[:provider] = %w[greenlight ldap google openid_connect].include?(user_hash[:provider]) ? 'greenlight' : user_hash[:provider]
+          # Re-write list of providers to greenlight
+          user_hash[:provider] = %w[greenlight ldap google openid_connect saml].include?(user_hash[:provider]) ? 'greenlight' : user_hash[:provider]
```

Comment:
Now that SAML is in this list, the V2 migration task should work without modification. Beforehand I had a modification in place which did match the provider to greenlight already in the migration task.